25049.2 <c-60#127> Created logger
25049.2 <c-60#127> appl_initialize was called
25049.2 <c-60#127:9> Logger was updated
25049.2 <c-60#127:9> Started intializing cube lib
25049.2 <c-60#127:9> Finished intializing cube lib with worker, starting to work
25049.2 <c-60#127:9> requestCubes => [41] Buffer size: 0
25049.2 <c-60#127:9> Sent requestCubes signal to root
25049.2 <c-60#127:9> Received 64 cubes from root
25049.2 <c-60#127:9> The main loop continues
25049.2 <c-60#127:9> Started solving a cube
25049.6 <c-60#127:9> Cube failed
25049.6 <c-60#127:9> Added failed cube
25049.6 <c-60#127:9> Started solving a cube
25049.6 <c-60#127:9> Cube failed
25049.6 <c-60#127:9> Added failed cube
25049.6 <c-60#127:9> Started solving a cube
25049.6 <c-60#127:9> Cube failed
25049.6 <c-60#127:9> Added failed cube
25049.6 <c-60#127:9> Started solving a cube
25049.6 <c-60#127:9> Skipped cube
25049.6 <c-60#127:9> Started solving a cube
25049.9 <c-60#127:9> appl_interrupt was called
25049.9 <c-60#127:9> Interrupting worker
25049.9 <c-60#127:9> appl_interrupt was called
25049.9 <c-60#127:9> appl_withdraw was called
25049.9 <c-60#127:9> Started withdraw thread
25049.9 <c-60#127:9> STOP (0.00s since last cb)
25049.9 <c-60#127:9> Solving interrupted
25049.9 <c-60#127:9> The main loop continues
25049.9 <c-60#127:9> Leaving the main loop due to an interruption
25049.9 <c-60#127:9> Finished withdrawing lib
25049.9 <c-60#127:9> Enter destructor of CubeWorker
25049.9 <c-60#127:9> Time waiting for messages: 0.000
25049.9 <c-60#127:9> Enter destructor of CubeWorkerInterface
25049.9 <c-60#127:9> Enter destructor of CubeRoot
25049.9 <c-60#127:9> Finished withdraw thread
25110.3 <c-60#127:9> Enter destructor
25110.3 <c-60#127:9> Time working: 0.672
25110.3 <c-60#127:9> Time suspended: 0.000
25110.3 <c-60#127:9> Joined cleanup thread
25110.3 <c-60#127:9> Exit destructor
25110.3 <c-60#127:9> Destructing logger
